app/views/internal/configs/show.html.erb

```diff
@@ -679,6 +679,26 @@
           </div>
         </div>
 
+        <div class="card mt-3">
+          <%= render partial: "card_header",
+                     locals: {
+                       header: "Sponsors",
+                       state: "collapse",
+                       target: "sponsorsContainer",
+                       expanded: "false"
```

Comment:
Any reason we need a `state` and `expanded` property in locals? They seem like they would so the same thing.

Reply:
Yeah, I dunno. This is the pattern we have established across all the configs. You're probably right, but refactoring this would be out-of-scope of the PR.